xds/internal/balancer/clusterresolver/clusterresolver.go

```diff
@@ -306,6 +310,11 @@ func (b *clusterResolverBalancer) run() {
 			switch update := u.(type) {
 			case *ccUpdate:
 				b.handleClientConnUpdate(update)
+				if update.done != nil {
+					// Close the channel to convey to the consumers of updateCh
+					// that child policies config are updated inline.
```

Comment:
// Close the channel to signal to consumers of updateCh
// that the child policies' configuration has been updated.

Reply:
Updated